geode-protobuf/src/test/java/org/apache/geode/protocol/protobuf/operations/PutAllRequestOperationHandlerJUnitTest.java

```diff
@@ -14,6 +14,14 @@
  */
 package org.apache.geode.protocol.protobuf.operations;
 
+import static org.mockito.Mockito.any;
```

Comment:
spotlessApply doesn't seem to care about the import order, however IntelliJ will reorder according the style guide every time you run "optimize imports".  Patrick was flagging every pull request with the outdated order for a while.